modules/chapter1/pages/section2.adoc

```diff
@@ -2,14 +2,14 @@
 
 == A common machine learning workflow
 
-Data Preparation: Collecting, cleaning, and preprocessing the data. E.g removing outliers.
+Data Exploration and Preparation: Includes collecting, cleaning, and preprocessing the data. This means investigating, removing and/or substituting missing data points and outliers. Representation of occurances and labels is also investigated and visualized through different techinques.
 
-Feature Engineering: Selecting or creating relevant features to represent the data. E.g binning income.
+Feature Engineering: Selecting and/or creating relevant features to represent the data. One example is creating another feature to indicate whether an occurance is missing a variable or not, especially if it follows a pattner. Another is masking gender or transforming age into intervals in order to not discriminate towards age. Sometimes a feature may be deemed non-contributing and removed all together. 
```

Comment:
```suggestion
Feature Engineering: Selecting and/or creating relevant features to represent the data. One example is creating another feature to indicate whether an occurrence is missing a variable or not, especially if it follows a pattner. Another is masking gender or transforming age into intervals in order to not discriminate towards age. Sometimes a feature may be deemed non-contributing and removed all together. 
```

Reply:
Also, what does "pattner" mean? Did you mean "partner"?

Reply:
Ops, its should say pattern.